charter.html

```diff
@@ -277,6 +277,10 @@ <h2>Coordination</h2>
           such that the group is encouraged to coordinate with the appropriate groups at the WHATWG, Ecma, and IETF as needed.
         </p>
 
+		<p>
+			Acknowledging that private advertising technology has broad impact on a variety of less technical stakeholders, some who may not have access to the working group or may have less technical expertise than Working Group participants, the WG will coordinate with the Private Advertising Technology Community Group to set up limited periods to gather feedback. At any time, but always when a specification is undergoing a <a href="https://www.w3.org/2021/Process-20211102/#rec-track" title="The W3C Recommendation Track">state change</a>, the WG will ask the CG for feedback. The WG will allow the CG at least 4 weeks to gather that feedback. The goal is to solicit feedback where meaningful change to the document from the last time feedback was gathered occurs. This Working Group will initiate the feedback period by opening an issue <a href="https://github.com/patcg/admin" title="PAT CG Admin Repository">on the administration repository</a>. The Working Group is advised to alert PATCG at least 3 months before first entering <a href="https://www.w3.org/Consortium/Process/#RecsCR" title="Candidate Recommendation">CR</a> in order to allow for PATCG to actively collect more extensive feedback from the community and is encouraged to proactively notify PATCG when major changes occur in a specification following a review. 
```

Comment:
This could be even shorter, I think.

```suggestion
			The WG will coordinate with the Private Advertising Technology Community Group to solicit feedback from a wider community of interest. At any time, but always when a specification is undergoing a <a href="https://www.w3.org/2021/Process-20211102/#rec-track" title="The W3C Recommendation Track">state change</a>, the WG will ask the CG for feedback. The WG will allow the CG at least 4 weeks to gather that feedback.
			<p>The goal of coordinating with the CG is to solicit feedback where meaningful change to the document from the last time feedback was gathered occurs.
```

Reply:
This looks good to me!